docs/snippets/react/button-story-click-handler-args.js.mdx

```diff
@@ -1,9 +1,18 @@
 ```js
 // Button.stories.js | Button.stories.jsx | Button.stories.ts | Button.stories.tsx
 
-export const Text = ({ label, onClick }) => <Button label={label} onClick={onClick} />;
-Text.args = {
-  label: 'Hello',
-  onClick: action('clicked'),
+import React from 'react';
+
+import { action } from '@storybook/addon-actions';
+
+import { Button } from './Button';
```

Comment:
Are we wanting to commit to a named/default import for these examples? I've seen them switch a few times between examples and wasn't sure if we were looking for consistency here.

Reply:
@darleendenno thanks for raising that question. To give you some insights, the react example button is a named export, but Vue, Angular, Svelte are not. They are default exports. My thought process for snippets is the following:
- React use the named export
- Vue/Angular/Svelte (default export)
- Common snippets that reference the component default to React.

Let me know and we'll go from here.

Reply:
That sounds great! If you'd like, I will make some notes where I see inconsistencies. Thank you 💯 🎸